packages/commons-server/src/libs/server/crud.ts

```diff
@@ -53,13 +58,16 @@ const findItemIndex = (databucketValue: any, request: Request) =>
  * @param routePath
  * @returns
  */
-export const crudRoutesBuilder = (routePath: string): CrudRoutes => {
+export const crudRoutesBuilder = (
+  routePath: string,
+  routeCrudKey: string
```

Comment:
Remove param and "hardcode" the route param to `:id`, it's transparent for the user and will contribute to avoid a server restart when changing the `crudKey`